codalab/lib/bundle_cli.py

```diff
@@ -502,6 +544,10 @@ def do_command(self, argv):
         argv = self.collapse_bare_command(argv)
         args = parser.parse_args(argv)
 
+        # print args.help
```

Comment:
Ah ok, makes sense to me.
On Fri, Nov 6, 2015 at 6:01 PM Percy Liang <notifications@github.com> wrote:

> In codalab/lib/bundle_cli.py
> <https://github.com/codalab/codalab-cli/pull/205#discussion_r44205747>:
>
> > @@ -502,6 +544,10 @@ def do_command(self, argv):
> >          argv = self.collapse_bare_command(argv)
> >          args = parser.parse_args(argv)
> >
> > +        # print args.help
>
> I propose renaming wcp to wadd.
> … <#msg-f:1517144412292410190_>
> On Fri, Nov 6, 2015 at 5:54 PM, Stephen Koo <notifications@github.com>
> wrote: In codalab/lib/bundle_cli.py <#205 (comment)
> <https://github.com/codalab/codalab-cli/pull/205#discussion_r44205636>>:
> > @@ -502,6 +544,10 @@ def do_command(self, argv): > argv =
> self.collapse_bare_command(argv) > args = parser.parse_args(argv) > > + #
> print args.help remove — Reply to this email directly or view it on GitHub <
> https://github.com/codalab/codalab-cli/pull/205/files#r44205636>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/codalab/codalab-cli/pull/205/files#r44205747>.
>